### PR TITLE
Update instructions to use the last django version (1.8.2)

### PR DESCRIPTION
**Pull request by brouberol**
These changes have already been applied to the French version, in this pull request: https://github.com/Natim/django-carrots/pull/14

### CODE REVIEW

**Comment by plocharz-9livesdata** [comment]
Hi, thanks for an update. I'll probably look at this during the weekend.

**Comment by brouberol** [comment]
Cool, thanks.

--
B
 
 
On Tue, Jun 9, 2015, at 06:07 PM, Krzysztof Płocharz wrote:
> Hi, thanks for an update. I'll probably look at this during the weekend.


> —
> Reply to this email directly or view it on GitHub[1]. 

>  


Links:

  1. https://github.com/ggcarrots/django-carrots/pull/48#issuecomment-110417083

File: source/django_in_action.rst [hunk 1]
```diff
@@ -130,38 +130,40 @@ To make it simpler we also exclude the advanced support for time zones in the da
...
+     Applying admin.0001_initial... OK
+     Applying contenttypes.0002_remove_content_type_name... OK
+     Applying auth.0002_alter_permission_name_max_length... OK
+     Applying auth.0003_alter_user_email_max_length... OK
+     Applying auth.0004_alter_user_username_opts... OK
+     Applying auth.0005_alter_user_last_login_null... OK
+     Applying auth.0006_require_contenttypes_0002... OK
+     Applying sessions.0001_initial... OK
+   (workshops) ~/carrots$ python manage.py createsuperuser
+   Username (leave blank to use 'br'): admin
```
Comment (qrees) [inline source/django_in_action.rst]: Wrong username. There is "beans" in paragraph below.

File: source/django_models.rst [hunk 2]
```diff
@@ -22,23 +22,31 @@ In the file ``polls/models.py`` we type::
...
-   Installed 0 object(s) from 0 fixture(s)
-
-That’s it! However, probably we would like to be able to edit objects. The easiest way is to do it in 
+    (workshops) ~/carrots$ python manage.py makemigrations
+    Migrations for 'polls':
+      0001_initial.py:
+        - Create model Choice
+        - Create model Poll
+        - Add field poll to choice
+    (workshops) ~/carrots$ python manage.py migrate
```
Comment (qrees) [inline source/django_models.rst]: I'd split text here, so that it is visible that this is another command.

**Comment by brouberol** [comment]
I've applied the corrections you asked for.

**Comment by qrees** [comment]
Would you mind also rebasing on the ggcarrots:master, as this version won't merge cleanly?

**Comment by brouberol** [comment]
Done. Phew, that was a headache.

**Comment by qrees** [comment]
Great  work :smile:

**Pull request merged**